substrate/frame/multisig/src/benchmarking.rs

```diff
@@ -20,8 +20,7 @@
 #![cfg(feature = "runtime-benchmarks")]
 
 use super::*;
-#[allow(deprecated)]
-use frame::benchmarking::v1::*;
+use frame::benchmarking::v2::*;
```

Comment:
```suggestion
use frame_benchmarking::v2::*;
```

Reply:
Ah wait that's the umbrella crate. You should be able to use that too, `v2` doesn't exist at the top level though, I think you get it via the `prelude`

Reply:
fixed in https://github.com/paritytech/polkadot-sdk/pull/6018/commits/57828cd5f8fb60b691c260bd34a0da859469038c